clients/admin-ui/src/features/privacy-notices/form.ts

```diff
@@ -21,27 +25,25 @@ export const defaultInitialValues: PrivacyNoticeCreation = {
 
 export const transformPrivacyNoticeResponseToCreation = (
   notice: PrivacyNoticeResponse
-): PrivacyNoticeCreation => {
-  // Remove the fields not needed for editing/creation
```

Comment:
this is the second time this bug has come up! I thought adding a test would fix it, but it turns out not since the text fixture didn't have the new field 🥲 

so no more removing fields aka opting out—we're moving to opt in only! in other words, we create a new object from scratch, rather than starting with an original object and removing the fields we think we don't need.

Reply:
Good idea! opt-in should be more resilient